src/FixedDataTableRow.js

```diff
@@ -170,8 +170,12 @@ class FixedDataTableRowImpl extends React.Component {
       return true;
     }
 
-    // if row is not visible then no need to render it
-    // change in visibility is handled by the parent
+    // if row's visibility has changed, then update it
+    if (this.props.visible !== nextProps.visible) {
```

Comment:
Rerender the component when visibility changes, similar to the [usage here](https://github.com/schrodinger/fixed-data-table-2/blob/afbc448ba4dae70e9591e24e550f6cebc1081c12/src/FixedDataTableRow.js#L538-L546).